README.md

```diff
@@ -1,67 +1,69 @@
+Failure to abide by these rules may lead to being asked to leave FAC.
 
 ## FAQ
 
-**How much time do I need to give back each month?**    
-As a minimum, regardless of where you are based, you need to be active in at least one FAC study programme or meetup group. Members are expected to contribute one hour of support to FAC programmes for every day that they are working in the co-working space on their own projects. If you have just completed one of the FACx programmes, then it is your collective responsibility with the rest of your cohort to facilitate FACx+1.
+**How much time do I need to give back?**    
+As a minimum, regardless of where you are based, you need to be active in at least one FAC study programme or meetup group. If you have just completed one of the FACx programmes, then it is your collective responsibility with the rest of your cohort to facilitate FACx+1.
 
 **What if I do not feel confident being a mentor or running a study programme?**    
-Think about what you are most interested in and find another member who would be prepared to work with you to find ways to share your own learning. Think of where your strengths are and make a contribution accordingly.
+Think about what you are most interested in and find someone who would be prepared to work with you to find ways to share your own learning. Think of where your strengths are and make a contribution accordingly.
 
-**What is meant by an *agency relationship*?**    
-An agreement whereby FAC receives a fee from the employer. This fee is the main source of income for FAC and will ensure that we will be able to continue to offer the opportunities to others that our members have already benefited from.
+**What are *"recruitment fees"*? Do they come out of my salary if I get a job through FAC?**    
+FAC acts like a recruitment agency in finding work for graduates; employers pay a fee to FAC just as they would to a standard recruitment agency. This fee is the main source of income for FAC and will ensure that we will be able to continue to offer to others the opportunity that graduates of our programme have already benefited from. Recruitment fees are one-off lump sums usually expressed as a percentage of an employee's first year salary, but are paid directly by the employer and do not actually come out of the the employee's salary. 
 
 **What if I receive an unsolicited offer of work?**    
-Refer them to us and follow our guidelines on how to communicate with prospective employers. [in preparation, but likely to include a recommendation that all members put a note on their linkedin profiles that refers prospective employers to FAC]
+Refer them to us and follow our guidelines on how to communicate with prospective employers. It may be useful to display some expanatory text on your LinkedIn page or other professional platforms - FAC will provide this (but it is not mandatory).
```

Comment:
* Spelling mistake: **explanatory**
* Am I right in assuming these guidelines have yet to be written?  Would be awesome if instructions on where to find the guidelines were provided (once they exist :smile:)